### PR TITLE
#3 Changes to SelectorFilter and PeriodGranularity

### PR DESCRIPTION
**Pull request by paruljain21**
Added Integer as value in SelectorFilter
Added PeriodGranularity to extend Granularity

### CODE REVIEW

File: src/test/java/in/zapr/druid/druidry/granularity/PeriodGranularityTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package in.zapr.druid.druidry.granularity;
```
Comment (GG-Zapr) [inline src/test/java/in/zapr/druid/druidry/granularity/PeriodGranularityTest.java]: Please add license in the file

File: src/main/java/in/zapr/druid/druidry/granularity/PeriodGranularity.java [hunk 2]
```diff
@@ -39,5 +43,13 @@
 
     // TODO: Check if it is alright
     private DateTimeZone timeZone;
-    private Date origin;
+    private DateTime origin;
+
+    public String getOrigin() {
```
Comment (GG-Zapr) [inline src/main/java/in/zapr/druid/druidry/granularity/PeriodGranularity.java]: Why is getter returning String?

Comment (paruljain21) [reply src/main/java/in/zapr/druid/druidry/granularity/PeriodGranularity.java]: It has to. In JSON to druid we send String only. If we don't return String, the origin will return "origin":{"weekyear":2017,"yearOfEra":2017,"millisOfDay":68420218,"monthOfYear":11,"hourOfDay":19,"minuteOfHour":0,"secondOfMinute":20,"millisOfSecond":218,"weekOfWeekyear":46,"yearOfCentury":17,"centuryOfEra":20,"secondOfDay":68420,"minuteOfDay":1140,"year":2017,"dayOfMonth":13,"dayOfWeek":1,"era":1,"dayOfYear":317,"chronology":{"zone":{"fixed":false,"uncachedZone":{"fixed":false,"cachable":true,"id":"Asia/Kolkata"},"id":"Asia/Kolkata"}},"zone":{"fixed":false,"uncachedZone":{"fixed":false,"cachable":true,"id":"Asia/Kolkata"},"id":"Asia/Kolkata"},"millis":1510579820218,"afterNow":false,"beforeNow":true,"equalNow":false}

File: src/test/java/in/zapr/druid/druidry/granularity/PeriodGranularityTest.java (hunk 1 above)
Comment (paruljain21) [reply src/test/java/in/zapr/druid/druidry/granularity/PeriodGranularityTest.java]: We do not have license in any of the Test file. Why would it be needed?

**Comment by GG-Zapr** [comment]
PR For #3

**Pull request merged**